programs/dibio.c

```diff
@@ -88,6 +89,18 @@ static UTIL_time_t g_displayClock = UTIL_TIME_INITIALIZER;
 #undef MIN
 #define MIN(a,b)    ((a) < (b) ? (a) : (b))
 
+/**
+  Returns the size of a file.
+  If error returns 0. Zero filesize or error is same for us.
+  Emit warning when the file is inaccessible or zero size.
```

Comment:
nit: This comment is out of date, or I'm missing something, I don't see any warnings emitted when this function returns 0.